ui/app/src/main/java/com/example/fuel/repository/UserRepository.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.fuel.repository
+
+import com.example.fuel.api.RetrofitInstance
+import com.example.fuel.model.User
+import com.example.fuel.viewmodel.UserViewModel
+import retrofit2.Response
+import retrofit2.http.Body
+
+class UserRepository {
+
+    suspend fun postLogin(user: User): Response<User> {
+        return RetrofitInstance.accountApi.postLogin(user.username, user.password)
+    }
+
+    suspend fun postRegister(user: User): Response<User> {

```

Comment:
Same here, you will receive 
![image](https://user-images.githubusercontent.com/61120635/201300934-eeecf5dc-5787-4c31-a5db-96d9a4c8066c.png)